vm/src/obj/objlist.rs

```diff
@@ -162,10 +162,27 @@ fn reverse(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
     }
 }
 
+pub fn contains(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
+    trace!("list.len called with: {:?}", args);
+    arg_check!(
+        vm,
+        args,
+        required = [(list, Some(vm.ctx.list_type())), (x, None)]
+    );
+    for element in get_elements(list).iter() {
+        if x == element {
```

Comment:
Instead of equals you can use here `call_method('__eq__')` or `call_method('__is__')` whichever is appropriate. Since the `==` operator will do a rust equals, I think this is not desirable.